### DISCUSSION TITLE
how-can-we-make-the-sharepoint-world-a-better-world/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#28] [Blog comments]
# how-can-we-make-the-sharepoint-world-a-better-world/

https://blogs.taiga.nl/martijn/posts/how-can-we-make-the-sharepoint-world-a-better-world/

**Comment by disqus-comments-migrator** [comment]
<em>Itamar commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at februari 26 2009, 06:50</em>

---

<p>a + b. unless you can get a big community around any changes you make, the project will be swallowed up when microsoft releases their next version and you will have to start using their new system. on the other hand, how long until next sharepoint is released?</p>

**Comment by disqus-comments-migrator** [comment]
<em>lee commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at maart 02 2009, 03:59</em>

---

<p>I feel your pain! Even creating the simplest webpart takes forever. When a client now wants something complex in sharepoint, we tend to fake it by spoofing the interface and using some cunning URL rewritting.</p>
